native/src/navigation/createNavigate.ts

```diff
@@ -47,17 +46,11 @@ const createNavigate =
       })
 
       if (routeInformation.route === LANDING_ROUTE) {
-        navigateToLanding({
```

Comment:
I think these changes should be reverted as they don't matter for your logic anyway right? Not sure but I have a fear that this will break deep linking otherwise.

Reply:
how is deep linking connected to the landing page? but i can revert that sure